tests/test_tekton.py

```diff
@@ -1209,3 +1209,21 @@ def custom_watch(api_path, api_version, resource_type, resource_name,
                         (task_run_3[TASK_RUN_NAME3]['pipelineTaskName'], '3Hello World'),
                         (task_run_3[TASK_RUN_NAME3]['pipelineTaskName'], '3'),
                         (task_run_3[TASK_RUN_NAME3]['pipelineTaskName'], '3Bye World')]
+
+
+def test_get_sorted_task_runs():
+    task_runs = {
+        "TaskRunSecond": {"status": {"startTime": "2022-09-07T18:23:51Z"}},
+        "TaskRunThird": {"status": {"startTime": "2022-09-07T18:24:51Z"}},
+        "TaskRunFirst": {"status": {"startTime": "2022-09-07T18:22:51Z"}},
+        "TaskRunMissingKey": {"status": {}},
+    }
+    expected_sorted = [
+        ("TaskRunFirst", {"status": {"startTime": "2022-09-07T18:22:51Z"}}),
+        ("TaskRunSecond", {"status": {"startTime": "2022-09-07T18:23:51Z"}}),
+        ("TaskRunThird", {"status": {"startTime": "2022-09-07T18:24:51Z"}}),
+        ("TaskRunMissingKey", {"status": {}}),
```

Comment:
It might be good to have at least two task runs with missing startTime in the unit test. I think it will still work fine but not 100% sure
```python
>>> (True, None) < (True, None)
False
```

Reply:
updated